include/z64camera.h

```diff
@@ -670,7 +670,7 @@ typedef struct {
 } KeepOn3ReadOnlyData; // size = 0x2C
 
 typedef struct {
-    /* 0x00 */ Vec3f eyeToAtTarget; // esentially a VecSph, but all floats.
+    /* 0x00 */ Vec3f eyeToAtTarget; // essentially a VecGeo, but all floats.
```

Comment:
This can be three floats too, why not `f32 eyeToAtTargetGeoR, eyeToAtTargetGeoYaw, eyeToAtTargetGeoPitch` or smth